pkg/pool-weighted/foundry.toml

```diff
@@ -29,11 +29,11 @@ evm_version = 'cancun'
 ignored_error_codes = [2394, 5574, 3860] # Transient storage, code size
 
 [fuzz]
-runs = 10000
+runs = 100
```

Comment:
Is this related, or left over from something else?

Reply:
I'd revert this one, seems unrelated.